Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -2719,9 +2719,9 @@ private void handleNewRequest(ServerRequest req) {
             if((req instanceof ServerRequestRegisterClose)){
                 Log.i(TAG, "Branch is not initialized, cannot close session");
                 return;
-            }
-            else{
-                initializeSession(null);
+            } else {
+                boolean isReferrable = prefHelper_.getIsReferrable() == 1;
```

Comment:
This case will occur when user calling a Branch API (other than open/install) when Branch is not initialized. At this point we have no idea what user want.
- In case app init session and it failed , this will use the user specified value. 
- in case app is trying to call any API before init  app will use last known value
- On a fresh install and API is called before init , this will use default value which is `non Referrable`